examples/polynomial/stages.yml

```diff
@@ -18,3 +18,21 @@
 
   script:
     - poly --coefficients coefficients.json --output output.json
+
+
+-
+  name: evaluate_polynomial_function
+  environment: polynomial
+
+  transportable_functions:
+    -
+      name: polynomial
+      args: coefficients.json
+
+  output:
+    -
+      record: polynomial_function_output
+      location: output_function.json
+
+  script:
```

Comment:
If a function step is to be a minimal chunk of work, do we need it to support multiple function calls? Should we consider letting a `function_step` be a single Python function and then if one wants to combine multiple functions that is done via the stages and the forward model?

Reply:
> Aha, so is it then so that if the output record name of one step matches the input record name of another step then the data will be forwarded from the first step to the other?

Yes, that's how it works now.